index.js

```diff
@@ -99,4 +99,34 @@ persistReconciliationText = async function (handle) {
   }
 }
 
-module.exports = { createNewTemplateFolder, importNewTemplateFolder, constructReconciliationText, persistReconciliationText }
+runTests = async function (handle) {
+  const relativePath = `./${handle}`
+  const config = fsUtils.readConfig(relativePath)
+  const testPath = `${relativePath}/${config.test}`
+  const testContent = fs.readFileSync(testPath, 'utf-8')
+
+  const testParams = { 'template': constructReconciliationText(handle), 'tests': testContent }
+
+  const testRunResponse = await api.createTestRun(testParams)
+  const testRunId = testRunResponse.data
+  let testRun = { 'status': 'started' }
+
+  while (testRun.status === 'started') {
+    await new Promise(resolve => setTimeout(resolve, 2000))
+
+    const response = await api.fetchTestRun(testRunId)
+    testRun = response.data
+  }
+
+  if (testRun.status === 'completed') {
```

Comment:
To avoid the nested if you can invert this condition to have a guard condition

Reply:
@ihcene could you please elaborate how can I invert it? I mean I just don't see it